modules/nf-core/ska/distance/meta.yml

```
@@ -0,0 +1,96 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "ska_distance"
+description: Calculate pairwise distances and basic clustering from SKA sketches
+keywords:
+  - genomics
+  - metagenomics
+  - kmer
+  - split_kmers
+  - distance
+  - clustering
+tools:
+  - "ska":
+      description: "SKA (Split Kmer Analysis)"
+      homepage: "https://github.com/simonrharris/SKA"
+      documentation: "https://github.com/simonrharris/SKA/wiki"
+      tool_dev_url: "https://github.com/simonrharris/SKA"
+      doi: "10.1101/453142"
+      licence: ['MIT']
+
+
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. `[ id:'sample1', single_end:false ]`
```

```suggestion
          e.g. `[ id:'sample1'  ]`
```
I think the single_end is not applicable here?